docs/source/onboarding_resources/computing/index.rst

```diff
@@ -280,22 +280,161 @@ lines from your trainings!
 Your Bash Configuration Files 
 -----------------------------
 
-Bash files contain commands you would enter into a command line, but 
-specifically ones you will use very often. For example, every time you 
-want to open a Jupyter session, the cluster needs certain information 
-and requirements. To find this information, it looks in your Bash files 
-rather than asking you to enter the same information every time. 
-
-However, Bash files can be confusing since it is less obvious when the 
-information is being used or what it is used for. Therefore, we have 
-provided a copy-and-paste formatting for information to be added to 
-your Bash files. 
-
-.. todo::
-
-  - Confirm and paste in bash files 
-  - Get Zeb's help in explaining Bash files better 
-  - Moving up since this is needed for a lot of things below 
+Bash configuration files contain information and commands that are used when 
+interacting with the cluster. When the cluster tries to execute some command 
+line prompts, it will look to your bash configuration files for information 
+or filepaths. 
+
+You should have 2 bash configuration files, your bash profile or :code:`.bash_profile` 
+and your bash rc or :code:`.bashrc`. 
+
+bash profile
+~~~~~~~~~~~~
+
+The bash profile is only run when you open the cluster, and so is usually a shorter file. 
+Below are the lines of code you need in your bash profile. 
+
+:: 
+
+  [[ -e ~/.profile ]] && source ~/.profile    ##Loads generic profile settings 
+  [[ -e ~/.bashrc  ]] && source ~/.bashrc     ##Loads bash rc 
+
+bash rc
+~~~~~~~
+
+Your bash rc file is run more frequently and can contain helpful settings and information 
+needed to run certain commands. First, you will need to run the below line of code in 
+your terminal to set up conda. Once you're logged into the cluster on the terminal of 
+your choosing, run: 
+
+.. code-block:: bash 
+  :linenos:
+
+  $ /ihme/code/central_comp/miniconda/bin/conda init  
```

Comment:
This is also mentioned above, in the conda section -- not necessarily a bad thing, but you might want to have some kind of call-out between the two places.

Reply:
Good call out!